### PR TITLE
The native client can navigate to backyard articles through the side menu

### PR DESCRIPTION
**Pull request by arunbhalli**
[*PT-Story:**  ](https://www.pivotaltracker.com/story/show/178279161)
### User Story 
``` 
As an App
In order to navigate to backyard articles
I want a side-bar navigation
``` 
## Changes proposed in this pull request: 
*  Drawer navigation
*  Multiple Stack navigators
*  Get location with Native
* Fetches backyard articles

## What I have learned working on this feature: 
* Location with Native & Expo
* Drawer navigation

## Screenshots (Client) 
image

image

### CODE REVIEW

File: views/MainView.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const MainView = ({ navigation }) => {
               );
             } else {
               return (
-                <Article article={item} key={item.id} navigation={navigation} />
+                <Article article={item} navigation={navigation} />
```
Comment (offcast-dev) [inline views/MainView.jsx:37]: Why did you remove key? Dosn't it raise a warning now?

**Review by offcast-dev — APPROVED** [review]

File: components/BackyardStack.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const Stack = createStackNavigator();
+
+const BackyardStack = () => {
+  return (
+    <Stack.Navigator
+      screenOptions={() => ({
+        headerStyle: {
+          backgroundColor: '#111518',
+        },
+        headerTintColor: '#CEC269',
```
Comment (DavDan1) [inline components/BackyardStack.jsx:16]: are't you giving const style  in  the end maybe? coz you did in other files?

File: components/HomeStack.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          title: 'FAKE ? NEWS',
+          headerTitleAlign: 'center',
+          headerTitleStyle: {
+            fontWeight: 'bold',
+          },
+          headerLeft: () => (
+            <AntDesign
+            testID='drawer-menu'
+              name='menu-fold'
+              style={{ color: '#CEC269', paddingLeft: 15 }}
```
Comment (DavDan1) [inline components/HomeStack.jsx:37]: same here

**Review by DavDan1 — APPROVED** [review]

File: views/MainView.jsx (hunk 1 above)
Comment (LucasKnudsen) [reply views/MainView.jsx:37]: Nops, FlatList takes care of that :)

File: components/BackyardStack.jsx (hunk 2 above)
Comment (LucasKnudsen) [reply components/BackyardStack.jsx:16]: I don't really see them as opposing methods. If it's not reused and only takes up one line, there's no need for three extra lines in the bottom. IMO I guess :P

File: components/HomeStack.jsx (hunk 3 above)
Comment (LucasKnudsen) [reply components/HomeStack.jsx:37]: Same as above
